<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WedgeDialog</class>
 <widget class="QDialog" name="WedgeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: rgba(255, 255, 255, 0);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="selectTab">
      <attribute name="title">
       <string>Select</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Serial</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="dirListWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Wedge</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="fileListWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Information</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>Ultrasonic phased array wedge family.</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="okPushButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelPushButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="defineTab">
      <attribute name="title">
       <string>Define</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1,0">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Wedge</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="defineListWidget"/>
         </item>
         <item>
          <widget class="QPushButton" name="delPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,0,0">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="modelLabel">
               <property name="text">
                <string>Model</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="serialLabel">
               <property name="text">
                <string>Serial</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="typeLabel">
               <property name="text">
                <string>Wave Type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="angleLabel">
               <property name="text">
                <string>Angle</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="delayLabel">
               <property name="text">
                <string>Delay</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="OrientationLabel">
               <property name="text">
                <string>Orientation</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Velocity</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="primaryOffsetLabel">
               <property name="text">
                <string>Primary Offset</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sndOffsetLabel">
               <property name="text">
                <string>Secondary Offset</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="HeightLabel">
               <property name="text">
                <string>Height</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="LengthLabel">
               <property name="text">
                <string>Length</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="refPointLlabel">
               <property name="text">
                <string>Ref Point</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLineEdit" name="modelLineEdit">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="serialLineEdit">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="typeComboBox">
               <item>
                <property name="text">
                 <string>L</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>S</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="angleDoubleSpinBox">
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="suffix">
                <string> °</string>
               </property>
               <property name="minimum">
                <double>0.500000000000000</double>
               </property>
               <property name="maximum">
                <double>20.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="delaySpinBox">
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="suffix">
                <string> us</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>1024</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="orientationComboBox">
               <item>
                <property name="text">
                 <string>Normal</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Revesal</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="velocityDoubleSpinBox">
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="value">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="primaryOffsetDoubleSpinBox">
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="sndOffsetDoubleSpinBox">
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="heightDoubleSpinBox">
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="lengthDoubleSpinBox">
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="refPointDoubleSpinBox">
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="suffix">
                <string> mm</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="saveApplyPushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>Save &amp;&amp; Apply</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="savePushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cancelDefinePushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
